packages/components/src/components/tick-progress/tick-progress.tsx

```diff
@@ -12,7 +23,7 @@ const Tick = ({ is_on }) => {
     );
 };
 
-const TickProgress = ({ className, rows, columns, size, value }) => {
+const TickProgress = ({ className, rows = 2, columns = 5, size = 10, value = 0 }: Partial<TTickProgress>) => {
```

Comment:
Why didn't use `?` for optional properties in `TTIckProgress` instead of `Partial`?

Reply:
Since the entire list of props is optional, I chose wrapping the type with Partial interface, rather than adding `?`